core/src/main/scala/com/nvidia/spark/rapids/tool/tuning/AutoTuner.scala

```diff
@@ -528,7 +533,13 @@ class AutoTuner(
     } else {
       // Add a warning comment indicating that the current setup is not optimal
       // and no memory-related tunings are recommended.
-      Left(autoTunerConfigsProvider.notEnoughMemComment(executorHeap + minOverhead))
+      // TODO: For CSPs, we should recommend a different instance type.
+      val minTotalExecMemRequired = (
+        // Estimate total executor memory by reversing the memory fraction adjustment.
+        // This accounts for memory reserved by the container manager (e.g., YARN).
+        (executorHeap + minOverhead + offHeapMemSetByUser) / platform.fractionOfAvailableMemory
```

Comment:
should be multiplied or divided? the GDOc referenced in the comments multiply `0.8 * total memory`